notebooks/run_model_on_snellius.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        result = subprocess.run([f"exe/STEMMUS_SCOPE {path_to_config}", "&gt;&gt;", f"{station_name}.out"], cwd = root_path_to_model, shell=True)
</pre><p>we can remove creating the <code>.out </code>file. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/processing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>